input/fsh/codesystems/CodeSystemLocalComponentCodes.fsh

```diff
@@ -0,0 +1,29 @@
+CodeSystem: LocalComponentCodesCS
```

Comment:
I think we need to change the name and description of this codesystem.
It started as just the names of observation components.
Now it is a collection of local codes used for various purposes within the IG.
Perhaps just LocalCodes

Reply:
Agree - would we want to call it something like localBFDRCodes in case birth defects IG ever depends on it to avoid confusion?